.eslintrc.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         "no-unused-vars": [2, { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" }],
     "indent": [2, 4, { "SwitchCase": 1 }],
     "max-len": [2, 120, 4],
-    "comma-dangle": [2, "never"],
+    "comma-dangle": [0, "never"],
```

Comment:
Lenge siden jeg jobbet med eslint nå, men vil ikke dette skru av regelen?

Reply:
I Tslint var regelen `"trailing-comma": [true, "never"]`, som da krever at det skal være et trailing comma. Tenker du det er best at vi setter `comma-dangle": [2, "never"]` å da ikke tillater det ? Siden koden er `>=ES5` er det ingen har det ingen betydning for funskjonalitet såvidt jeg vet. 
Var forresten `"comma-dangle": "off"` som førte til problemer i forrige commit, da `"off"` tydeligvis ikke er gyldig, men `[0, "never"]` er gyldig

Reply:
Lurer på om `true/false` som første argument i tslint styrer hvorvidt regelen skal være aktiv? :/ Og andre argument er konfigurasjon til den da. :/ Om det stemmer vil nok eslint varianten være `[2, "never"]`, evt `["error", "never"]` om det fungerer. Men eslint-config har jeg aldri blitt klok på, noen ganger er `off` greit andre ganger må man skrive `[0]`

Funksjonalitetsmessig har det fint lite å si ja, blir uansett transpilert bort av babel. Så vi er nok langt over i det subjektive område av kodekvalitet. :P Personlig har jeg aldri likt dangling-commas (derfor de er eksplisitt skrudd i dette repoet), men det har bare fordi jeg syns det gjør koden vanskeligere å lese og ser helt forferdelig ut. :P

Reply:
Tolker dokumntasjonen slik at `true/false` betyr `requires/disallows` trailing comma. https://palantir.github.io/tslint/rules/trailing-comma/

Tenker vi ikke tillater trailing comma da, er ikke så stor fan av det heller 👍 Setter `comma-dangle": ["error", "never"]` i eslint og rydder opp i komponentene da det virker som mange av dem inneholder comma-dangle